### PR TITLE
website: Updated footer year

### PR DESCRIPTION
**Pull request by gretanausedaite**
## Changes

Updated Footer year to 2023.
image

## Testing

Open home page and scroll to the bottom.

## Docs

N/A

### CODE REVIEW

File: apps/website/src/components/utils/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+export default (props: React.ComponentProps<'footer'>) => {
+  const currentYear = new Date().getFullYear();
+  return <footer {...props}>© {currentYear} Bentley Systems, Inc</footer>;
```
Comment (mayank99) [inline apps/website/src/components/utils/Footer.tsx:8]: does it need to be a react component?

Comment (gretanausedaite) [reply apps/website/src/components/utils/Footer.tsx:8]: Do you want it in Astro or inline?

I did react because it's easy to include variables in html.

Comment (mayank99) [reply apps/website/src/components/utils/Footer.tsx:8]: well, we want components to be astro by default (just as easy to include variables in html), and only use react when astro doesn't work

Comment (gretanausedaite) [reply apps/website/src/components/utils/Footer.tsx]: Changed to Astro.

**Review by mayank99 — APPROVED** [review]

File: apps/website/src/components/utils/Footer.astro [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+const props = Astro.props;
+const currentYear = new Date().getFullYear();
+---
+
+<footer {...props}>© {currentYear} Bentley Systems, Inc</footer>;
```
Comment (mayank99) [inline apps/website/src/components/utils/Footer.astro:6]:
```suggestion
<footer {...props}>© {currentYear} Bentley Systems, Inc</footer>
```

**Review by adamhock — APPROVED** [review]

**Pull request merged**